app/cdap/components/ResourceCenterEntity/ResourceCenterPipelineEntity.tsx

```diff
@@ -0,0 +1,191 @@
+  });
+  const hydratorImportLink = window.getHydratorUrl({
+    stateName: hydratorLinkStateName,
+    stateParams: {
+      ...hydratorLinkStateParams,
+      resourceCenterId,
+    },
+  });
+
+  const createBtnHandler = () => {
+    PlusButtonStore.dispatch({
+      type: 'TOGGLE_PLUSBUTTON_MODAL',
+      payload: {
+        modalState: false,
+      },
+    });
+  };
+
+  const importBtnHandler = (event) => {
+    if (!objectQuery(event, 'target', 'files', 0)) {
+      return;
+    }
+
+    const uploadedFile = event.target.files[0];
+
+    if (uploadedFile.type !== 'application/json') {
+      onError(T.translate(`${PREFIX}.errorLabel`), T.translate(`${PREFIX}.nonJSONError`));
+      return;
+    }
+
+    const reader = new FileReader();
+    reader.readAsText(uploadedFile, 'UTF-8');
+
+    reader.onload = (evt) => {
+      let fileDataString = evt.target.result;
+      const error = validateImportJSON(fileDataString);
+      if (error) {
+        onError(T.translate(`${PREFIX}.errorLabel`), error);
+        return;
+      }
+      fileDataString = adjustConfigNode(fileDataString);
+
+      onError(null, null);
+      window.localStorage.setItem(resourceCenterId, String(fileDataString));
+      window.location.href = hydratorImportLink;
+    };
+  };
+
+  const [pullPipelineModalState, dispatch] = useReducer(reducer, defaultState);
+
+  const pullPipelineBtnHandler = () => {
+    dispatch({ type: 'SET_LOADING', payload: { loading: true } });
+    const namespace = getCurrentNamespace();
+    validateSourceControlManagement(namespace).subscribe(
+      () => {
+        dispatch({ type: 'TOGGLE_MODAL' });
+      },
+      (err) => {
+        dispatch({
+          type: 'SET_ERROR',
+          payload: {
+            error: T.translate('features.SourceControlManagement.invalidConfig', {
+              error: err.message,
+            }),
+          },
+        });
+        dispatch({ type: 'SET_LOADING', payload: { loading: false } });
+      },
+      () => {
+        dispatch({ type: 'SET_LOADING', payload: { loading: false } });
+      }
+    );
+  };
+
+  return (
+    <>
+      <div className="resourcecenter-entity-card resourcecenter-entity-card-pipeline">
+        <div className="image-content-container">
+          <div className="image-container">
+            <div className="entity-image">
+              <IconSVG name={iconClassName} />
+            </div>
+          </div>
+          <div className="content-container">
+            <div className="content-text">
+              <h4>{title}</h4>
+              <p>{description}</p>
+            </div>
+          </div>
+        </div>
+        <div className="buttons-container">
+          <PrimaryContainedButton
+            href={hydratorCreateLink}
+            id="create-pipeline-link"
+            className="btn btn-primary"
+            onClick={createBtnHandler}
+          >
+            {primaryActionLabel}
+          </PrimaryContainedButton>
+          <input
```

Comment:
why isn't this a material ui input?

Reply:
This is the way MUI documentation writes it